### PR TITLE
Add new option property for npm-license-crawler

### PR DESCRIPTION
**Pull request by mu-hun**
You can see noColor option at https://github.com/mwittig/npm-license-crawler/blob/b46507f/README.md#options

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [ ] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/npm-license-crawler/index.d.ts [hunk 1]
```diff
@@ -1,16 +1,18 @@
-// Type definitions for npm-license-crawler 0.1
+// Type definitions for npm-license-crawler 0.1.1
```
Comment (ffflorian) [inline types/npm-license-crawler/index.d.ts:1]: This won't work as DT only tracks major and minor versions of packages

**Review by ffflorian — CHANGES_REQUESTED** [review]

File: types/npm-license-crawler/index.d.ts (hunk 1 above)
Comment (x86chi) [reply types/npm-license-crawler/index.d.ts:1]: I thought it was the version of @types/npm-license-crawler. I've [reset to latest version.](https://github.com/mwittig/npm-license-crawler/releases/tag/V0.1.4)

File: types/npm-license-crawler/index.d.ts [hunk 2]
```diff
@@ -4,13 +4,15 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.2
```
Comment (x86chi) [inline types/npm-license-crawler/index.d.ts:5]: @ffflorian Is it still need header?

Comment (ffflorian) [reply types/npm-license-crawler/index.d.ts:5]: If it works without it, you can remove it.

**Review by ffflorian — APPROVED** [review]

**Comment by ffflorian** [comment]
Ready to merge

**Pull request merged**